tests/system/Validation/RulesTest.php

```diff
@@ -609,6 +609,67 @@ public function testIsUniqueIgnoresParams()
 
 	//--------------------------------------------------------------------
 
+	/**
+	 * @group DatabaseLive
+	 */
+	public function testIsUniqueIgnoresParamsAsVariable()
```

Comment:
This are already covered in testIsUniqueIgnoresParamsPlaceholders

Reply:
Removed, thanks.